src/index.js

```diff
@@ -319,13 +320,15 @@ export default class Atrament extends AtramentEventTarget {
 
         this.#mouse.set(x, y);
         this.#mouse.previous.set(newX, newY);
-        // Android Chrome sets pressure to constant 1 by default, which would break the algorithm
+        // Android Chrome sets pressure to constant 1 by default,
+        //   which would break the algorithm
         // We also handle the case when pressure is 0.
         this.#pressure = position.pressure === 1
           ? DEFAULT_PRESSURE
           : position.pressure || DEFAULT_PRESSURE;
       } else {
         this.#mouse.set(x, y);
+        this.#mouse.previous.set(x, y);
```

Comment:
AFAICT this is the actual fix, right? It's interesting because the ORs on L317-318 are there exactly to prevent drawing lines from (0,0). But now I see that `#pointerUp` also calls `draw` and that's probably where the unwanted lines are drawn. Thank you very much for spotting it!

Reply:
Yes, this is the fix. Sorry about all the incidental changes.

Reply:
no worries, I don't mind since you've explained your reasoning for them :)